src/compiler/marks.ts

```diff
@@ -31,7 +31,7 @@ export function compileMarks(model: Model): any[] {
       type: MARKTYPES_MAP[mark],
       from: {
         // from.data might be added later for non-facet, single group line/area
-        transform: [{type: 'sort', by: sortBy}]
+        transform: [{ type: 'sort', by: sortBy }]
```

Comment:
This rules is nice if computer automatically do for us, again if there is no nice way to fix this without screwing up other style (Running formatter for the whole file is generally risky), let's turn this off.